### PR TITLE
update Anakin doc for v1.2

### PR DESCRIPTION
**Pull request by chenjiaoAngel**

### CODE REVIEW

File: doc/fluid/menu.json [hunk 1]
```diff
@@ -0,0 +1,592 @@
+{
```
Comment (shanyi15) [inline doc/fluid/menu.json]: please delete this file because this is from preview tool

**Comment by shanyi15** [comment]
辛苦之后提交的时候合并为一个commit，因为每一次commit都会发送邮件到RD的邮箱～

**Comment by chenjiaoAngel** [comment]
haloed

**Review by shanyi15 — APPROVED** [review]
LGTM

**Pull request merged**